### DISCUSSION TITLE
ORFS Check antenna: Command terminated by signal 6

### DISCUSSION THREAD

**Discussion by marcopoles** [The-OpenROAD-Project/OpenLane#2021] [Q&A]
Hi, 
working on import foreign tech, I'm running gcd desing and the flow stops at antenna check with following output:
<img width="1190" alt="image" src="https://github.com/The-OpenROAD-Project/OpenLane/assets/24896919/af98437f-13d9-47f1-b477-4cb674c795bd">

Any suggestion about the problem origin?

Thanks,
Marco

**Comment by maliberty** [comment]
Its hard to know.  Is this something where a test case could be provided?

**Reply by marcopoles** [reply]
I suppose that the test case creation will include lib+lef, so I can't. 
I tryed the flow on a very simple design and it works (2 FFs+2 NOR).
The tech is tsmc 40nm CMN40ULP, in case you know someone who has already worked on it.

I will proceed by trial and error to understand which std cell is causing problems.

Thanks,
Marco

**Reply by maliberty** [reply]
You could also ignore antennas while doing bring up.  The other option would be a 3-way NDA with Precision Innovations so that we could see the problem.

**Reply by marcopoles** [reply]
Following your suggestion to comment check_antennas in global_route.tcl the flow arrives to end.
We are available to a 3-way NDA. Do I have to write directly to info@precisioninno.com? 
Thanks

**Reply by maliberty** [reply]
Sure we can start a discussion from there.
